src/test/java/core/basesyntax/service/implementation/ValidatorImplTest.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.service.implementation;
+
+import core.basesyntax.service.Validator;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ValidatorImplTest {
+    private static Validator validator;
+
+    @BeforeClass
+    public static void setUp() {
+        validator = new ValidatorImpl();
+    }
+
+    @Test
+    public void validate_validData_ok() {
+        boolean actual = validator.validate("b,banana,25");
+        Assert.assertTrue(actual);
+    }
+
+    @Test
+    public void validate_invalidOperation_notOk() {
+        boolean actual = validator.validate("a,banana,25");
+        Assert.assertFalse("Invalid operation", actual);
+    }
+
+    @Test
+    public void validate_emptyFruitName_notOk() {
+        boolean actual = validator.validate("b,,25");
+        Assert.assertFalse("Fruit name is empty", actual);
+    }
+
+    @Test
+    public void validate_invalidFruitName_notOk() {
+        boolean actual = validator.validate("b,@@@@,25");
+        Assert.assertFalse("Invalid fruit name", actual);
+    }
+
+    @Test
+    public void validate_invalidFruitQuantity_notOk() {
+        boolean actual = validator.validate("b,banana,not");
+        Assert.assertFalse("Invalid quantity", actual);
+    }
+
+    @Test
+    public void validate_NegativeQuantity_notOk() {
+        boolean actual = validator.validate("b,banana,-8");
+        Assert.assertFalse("Quantity is negative", actual);
+    }
+
+    @Test
+    public void validate_EmptyQuantity_notOk() {
```

Comment:
```suggestion
    public void validate_emptyQuantity_notOk() {
```
check all your method names - they should be consistent